README.md

```diff
@@ -102,6 +102,12 @@ include any necessary hooks in a file called `custom.py` for Python models or `c
   - `model` is the deserialized model loaded by DRUM or by `load_model` (if provided)
   - This hook is intended to apply transformations to the prediction data before making predictions. It is useful
   if DRUM supports the model's library, but your model requires additional data processing before it can make predictions.
+- The transform hook will behave differently when running with a `transform` target type:
+    - `transform(X: DataFrame, transformer: Any, y: Series) -> Tuple[DataFrame, Series]`
+        - `X` is the dataframe given to DRUM to make predictions on. If `y` is defined, it should contain the raw features to be transformed (omitting the target).
+        - `transformer` deserialized transformer loaded by DRUM or `load_model` 
+        - `y` defaults to None, and should not be used for unsupervised downstream problems. Contains the target labels. Transform can simply pass the raw values on, may transform the values (i.e perform a log transform), 
```

Comment:
I'm just confused about using the word "should" here. The users don't really have control about what gets passed  in.

Reply:
I'd rather see "will"